python/marvin/contrib/vacs/mangagalaxyzoo.py

```diff
@@ -0,0 +1,90 @@
+# !usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Modified from mangahi.py by Karen Masters. 
+
+from __future__ import print_function, division, absolute_import
+
+import numpy as np
+import astropy
+import astropy.units as u
+import marvin.tools
+from marvin.tools.quantities.spectrum import Spectrum
+from marvin.utils.general.general import get_drpall_table
+from marvin.utils.plot.scatter import plot as scatplot
+
+from .base import VACMixIn
+
+
+class GZVAC(VACMixIn):
+    """Provides access to the MaNGA Galaxy Zoo Morphology VAC.
+
+    VAC name: GZ
+
+    URL: link
+
+    Description: Returns Galaxy Zoo morphology for MaNGA galaxies
+
```

Comment:
Can you write a brief description for your VAC here, and change the URL link to point to the DR15 documentation page for your VAC?